wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping/en.md

```diff
@@ -33,16 +33,16 @@ Suggestions themselves should ideally consist of four main parts, though based o
 
 There are any number of ways to present your mod post, so feel free to experiment, but try to keep the visuals simple and formatted in a way that it is easy for the mapper and others to work with.
 
-## Responding to a Mod
+## Responding to a mod
 
 Someone just posted some suggestions on your map, great! - but it doesn't end there. For the record, it's necessary to give a proper response for two major reasons. Firstly, so that future mods can be tailored more specifically to your concept or style, resulting in fewer unhelpful suggestions. Secondly, just as a mod post *helps you improve your mapping ability*, providing a detailed mod response helps the modder improve their modding ability, meaning they gain experience and can provide more helpful and detailed mods in the future. You never know who you may need to ask for a mod next time around!
```

Comment:
hmm, something tells me that dash isn't correct here..

```suggestion
Someone just posted some suggestions on your map, great! However, it doesn't simply end there. For the record, it's necessary to give a proper response for two major reasons. Firstly, so that future mods can be tailored more specifically to your concept or style, resulting in fewer unhelpful suggestions. Secondly, just as a mod post *helps you improve your mapping ability*, providing a detailed mod response helps the modder improve their modding ability, meaning they gain experience and can provide more helpful and detailed mods in the future. You never know who you may need to ask for a mod next time around!
```

Reply:
I'll take your word for English punctuation